model_compression_toolkit/qat/keras/quantization_facade.py

```diff
@@ -217,37 +215,27 @@ def keras_quantization_aware_training_finalize(in_model: Model):
              >>> quantized_model = mct.keras_quantization_aware_training_finalize(quantized_model)
 
          """
-
         def _export(layer):
             if isinstance(layer, qi.KerasQuantizationWrapper):
+                # Activation quantizers
+                inferableActivationQuantizers = []
+                if layer.dispatcher.is_activation_quantization:
+                    for quantizer in layer.dispatcher.activation_quantizers:
+                        if isinstance(quantizer, qi.BaseKerasTrainableQuantizer):
+                            inferableActivationQuantizers.append(quantizer.convert2inferable())
+                    layer.dispatcher.set_activation_quantizers(inferableActivationQuantizers)
+
+                # Weight quantizers
+                inferableWeightQuantizers = {}
```

Comment:
Your naming convention is not correct

A variable should be only written in lower case with _ to separate different words.
Example:inferableWeightQuantizers -> inferable_weight_quantizers

From more detailed see https://peps.python.org/pep-0008/